terraform/modules/sns/variables.tf

```diff
@@ -0,0 +1,13 @@
+variable "bucket_suffix" {
```

Comment:
I would name this variable just `suffix` since in this context we use as for the topic name.

This leads to my other querstion: do we need the suffix at all? Do topic names need to be unique across all accounts? Otherwise you can just give it the same name in all environments.

Reply:
It's a good question. I guess we don't really, I was just trying to stick with what seemed like a convention (with S3, for example). Including the env in the name makes it slightly less convenient but slightly harder to succeed when accidentally doing things in the wrong environment. I'll remove it.

Reply:
For S3, bucket names have to be unique across all accounts. Other than that we only use suffixes for feature branch deployments. In the core repo everything will be deployed to the default workspace and their will be only one resource per account.